packages/cli/src/networks/index.ts

```diff
@@ -112,11 +113,23 @@ export async function fetchWeakSubjectivityState(
   config: IChainForkConfig,
   url: string
 ): Promise<TreeBacked<allForks.BeaconState>> {
+  let stateBytes: Buffer | undefined;
   try {
     const response = await got(url, {headers: {accept: "application/octet-stream"}});
-    const stateBytes = response.rawBody;
+    stateBytes = response.rawBody;
```

Comment:
Refactor to avoid let variables

```ts
const response = await got(url, {headers: {accept: "application/octet-stream"}}).catch(e => {
      throw new Error("Unable to fetch weak subjectivity state: " + (e as Error).message);
})
try {
      return getStateTypeFromBytes(config, stateBytes).createTreeBackedFromBytes(stateBytes);
} catch (e) {
  // archive test and rethrow error
}
```